core-bundle/src/Repository/RememberMeRepository.php

```diff
@@ -31,9 +31,7 @@ public function __construct(ManagerRegistry $registry)
     {
         parent::__construct($registry, RememberMe::class);
 
-        if (($connection = $registry->getConnection()) instanceof Connection) {
-            $this->connection = $connection;
-        }
+        $this->connection = $registry->getConnection();
```

Comment:
How is this related to the PR ?

Reply:
Woops, it is not. I accidentally changed this in the wrong repository.

Reply:
Nope, that was me. Sorry. The change should have been a reminder to fix this in a separate PR. 🙈